requirements.in

```diff
@@ -5,7 +5,7 @@ Django<5.0
 django-anymail
 django-better-admin-arrayfield
 django-click
-django-crispy-forms<2
+django-crispy-forms<3
```

Comment:
Weird, Dependabot is updating `.in` files

Reply:
I do both for several projects and it feels like both are fragile. I do like the batching but I'm open to better.